src/webots/gui/WbGuiApplication.cpp

```diff
@@ -479,8 +479,13 @@ bool WbGuiApplication::setup() {
   WbSysInfo::initializeOpenGlInfo();
   WbWrenOpenGlContext::doneWren();
 
-  if (showGuidedTour)
+
+  if (true) //(showGuidedTour)
     mMainWindow->showGuidedTour();
+  
+  if (true)//(prefs->value("Internal/firstLaunch", true).toBool()))
```

Comment:
The update dialog will be shown even if it is my first time using the software?

Reply:
Yes, it is not ideal but it seems that we don't have a flag to know if the user has updated or installed Webots.

Reply:
I added that the note is for previous users